addons/stock/models/stock_picking.py

```diff
@@ -74,6 +74,11 @@ def _default_show_operations(self):
         help="How products in transfers of this operation type should be reserved.")
     reservation_days_before = fields.Integer('Days', help="Maximum number of days before scheduled date that products should be reserved.")
     reservation_days_before_priority = fields.Integer('Days when starred', help="Maximum number of days before scheduled date that priority picking products should be reserved.")
+    packaging_reserve_method = fields.Selection([
```

Comment:
Could you move this part on product category instead of picking type (already checked with thd)